internal/pkg/cli/init.go

```diff
@@ -185,24 +233,32 @@ containerized services that operate together.`))
 	if err := o.loadApp(); err != nil {
 		return err
 	}
-	if err := o.loadSvc(); err != nil {
+
+	if err := o.loadWkld(); err != nil {
 		return err
 	}
+	if o.initWkldVars.wkldType == manifest.ScheduledJobType {
+		log.Infof("Ok great, we'll set up a %s named %s in application %s running on the schedule %s.\n",
+			color.HighlightUserInput(o.initWkldVars.wkldType), color.HighlightUserInput(o.initWkldVars.name), color.HighlightUserInput(o.initWkldVars.appName), color.HighlightUserInput(o.initWkldVars.schedule))
+	} else {
+		log.Infof("Ok great, we'll set up a %s named %s in application %s listening on port %s.\n",
+			color.HighlightUserInput(o.initWkldVars.wkldType), color.HighlightUserInput(o.initWkldVars.name), color.HighlightUserInput(o.initWkldVars.appName), color.HighlightUserInput(fmt.Sprintf("%d", o.initWkldVars.port)))
```

Comment:
nit: it's possible now that backend services don't have a port specified can we check if port is non 0 before writing it out.